server/schedulers/evict_slow_store.go

```diff
@@ -89,6 +89,20 @@ func (conf *evictSlowStoreSchedulerConfig) getSchedulerName() string {
 	return EvictSlowStoreName
 }
 
+func (conf *evictSlowStoreSchedulerConfig) getStores() []uint64 {
+	// TODO: shoud we add a lock?
+	return conf.EvictedStores
+}
+
+func (conf *evictSlowStoreSchedulerConfig) getKeyRangesByID(id uint64) []core.KeyRange {
+	for _, storeID := range conf.EvictedStores {
```

Comment:
no need for loop for now I think

Reply:
yes, we do not need a loop,  furthermore, we do not need a slice for `EvictedStores` too. However, with a slice of `EvictStores` , I think a loop may cause less confusion. Here the usage of the loop is cheap so we needn't consider the performance here.